irohad/execution/impl/query_execution.cpp

```diff
@@ -251,18 +251,18 @@ QueryProcessingFactory::executeGetAccount(
 QueryProcessingFactory::QueryResponseBuilderDone
 QueryProcessingFactory::executeGetAccountAssets(
     const shared_model::interface::GetAccountAssets &query) {
-  auto acct_asset =
-      _wsvQuery->getAccountAsset(query.accountId(), query.assetId());
+  auto acct_assets =
+      _wsvQuery->getAccountAssets(query.accountId());
 
-  if (not acct_asset) {
+  if (not acct_assets) {
     return buildError<shared_model::interface::NoAccountAssetsErrorResponse>();
   }
-
-  const auto &account_asset = **acct_asset;
+  std::vector<shared_model::proto::AccountAsset> account_assets;
+  for (auto asset: *acct_assets)
+    account_assets.push_back(
+        *std::static_pointer_cast<shared_model::proto::AccountAsset>(asset));
```

Comment:
Please leave TODO to remove static cast when query response builder is updated and accepts interface objects